### PR TITLE
Create initial docs site

### PR DESCRIPTION
**Pull request by itskitto**
## Changes

Adds a new directory that houses a docs site as discussed in the Discord
image

All of the docs will still need to be added, I've pushed with only these additions as to provide others opportunity to step in and help with converting and structuring the docs to the docs site.

## Testing

- [X] Tests are passing
- [ ] Tests updated where necessary

## Docs

N/A

### CODE REVIEW

File: docs-www/src/layouts/Main.astro [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+            </button>
+            <a id="site-title" href="/">
+              <svg width="1em" height="1em" viewBox="0 0 340 340" fill="none" xmlns="http://www.w3.org/2000/svg">
+                <path d="M320 170C320 170 296.88 171.746 267.428 188.75C237.975 205.754 205.754 237.975 188.75 267.428C171.746 296.88 170 320 170 320C170 320 168.254 296.88 151.25 267.428C134.246 237.975 102.025 205.754 72.5721 188.75C43.1197 171.746 20 170 20 170C20 170 43.1197 168.254 72.5721 151.25C102.025 134.246 134.246 102.025 151.25 72.5721C168.254 43.1197 170 20 170 20C170 20 171.746 43.1197 188.75 72.5721C205.754 102.025 237.975 134.246 267.428 151.25C296.88 168.254 320 170 320 170Z" fill="currentColor"/>
+              </svg>
+<<<<<<< HEAD
+              <h1>Astroid</h1>
+=======
+              <h1>Astro Docs</h1>
+>>>>>>> f03d9a52edc0abba60d64a48f6a5456bdff3388f
```
Comment (matthewp) [inline docs-www/src/layouts/Main.astro:202]: Woops, mind fixing this?

File: docs-www/src/pages/index.astro [hunk 2]
```diff
@@ -0,0 +1,68 @@
+---
+import { Markdown } from 'astro/components';
+import Layout from '../layouts/Main.astro';
+---
+
+<<<<<<< HEAD
+<Layout content={{ title: 'Astro Documentation' }}>
+=======
+<Layout content={{ title: 'Astro Docs' }}>
+>>>>>>> f03d9a52edc0abba60d64a48f6a5456bdff3388f
```
Comment (matthewp) [inline docs-www/src/pages/index.astro:10]: This too

**Review by matthewp — CHANGES_REQUESTED** [review]
Looks like a couple of merge conflicts are in.

File: docs-www/src/layouts/Main.astro (hunk 1 above)
Comment (itskitto) [reply docs-www/src/layouts/Main.astro]: Woops indeed! Fixed.

File: docs-www/package.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  "private": true,
+  "scripts": {
+    "start": "astro dev",
+    "build": "astro build"
+  },
+  "devDependencies": {
+    "astro": "^0.13.0"
+  },
+  "snowpack": {
+    "workspaceRoot": "../.."
```
Comment (matthewp) [inline docs-www/package.json:13]: I think this should be `".."`.

**Comment by matthewp** [comment]
Can you remove the `docs-www/yarn.lock `? If you add `docs-www` to the package.json `workspaces` field then you can just run `yarn` from the root directory and everything should be linked up right.

**Comment by itskitto** [comment]
> Can you remove the `docs-www/yarn.lock `? If you add `docs-www` to the package.json `workspaces` field then you can just run `yarn` from the root directory and everything should be linked up right.

Gotcha! Made those changes, ran tests and also started up the dev server from `docs-www/` and everything worked as expected.

**Review by matthewp — APPROVED** [review]
Looks good, thanks!

**Pull request merged**